lambda/custom/index.js

```diff
@@ -120,13 +138,52 @@ function handleUserGuess(userGaveUp, handlerInput) {
   }
 
   // Check if we can exit the game session after GAME_LENGTH questions (zero-indexed)
+  
   if (sessionAttributes.currentQuestionIndex === GAME_LENGTH - 1) {
+    
+    speechOutput_APL_1 = speechOutput + speechOutputAnalysis
+    speechOutput_APL_2 = requestAttributes.t(
+      'GAME_OVER_MESSAGE',
+      currentScore.toString(),
+      GAME_LENGTH.toString()
+    ); 
     speechOutput = userGaveUp ? '' : requestAttributes.t('ANSWER_IS_MESSAGE');
     speechOutput += speechOutputAnalysis + requestAttributes.t(
       'GAME_OVER_MESSAGE',
       currentScore.toString(),
       GAME_LENGTH.toString()
     );
+    
+    
+
+
+    
+    if (supportsDisplay(handlerInput)) {
+      MAIN_DATA_SOURCE.phrase.teaser = speechOutput_APL_1;
+      MAIN_DATA_SOURCE.phrase.properties.phraseSI = '<speak>' + speechOutput_APL_1 + '</speak>';
+      MAIN_DATA_SOURCE.nextPhrase.properties.nextPhraseSI = '<speak>' + speechOutput_APL_2 + '</speak>';
+      MAIN_DATA_SOURCE.nextPhrase.teaser = speechOutput_APL_2;
+      speechOutput = "";
```

Comment:
This approach seems messy to me. I would rather see the entirety of the datasource in here rather than modifying the imported object. You can encapsulate it in a function or factory if you are worried about index.js size.

Reply:
That was originally in the code and was asked as part of all APL structure to be removed from index.js